packages/codepen-loader/CHANGELOG.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "@uifabric/codepen-example-loader",
```

Comment:
I think we should match your package name with the folder. I like how this is more specific as a name. Maybe rename your folder to `codepen-example-loader` would be appropriate.

Reply:
I renamed the package to `@uifabric/codepen-loader` because even that is obnoxiously long to type as a loader name inline (see comment in PR description)... But if you feel strongly about the name I can make it longer again.

Reply:
Discussed on chat and we decided to leave the name as-is.